### ISSUE TITLE
hahm.set_variable_value not working

### ISSUE THREAD

**Issue by Lemocin** [SukramJ/hahomematic#164]
**custom_component/hahomematic version (if applicable):**
0.18.0 and 0.19.0

**Home Assistant version (if applicable):**
2021.12.9

**CCU version:**

    Model (CCU2/3, Raspi3/4, NUC, ProxmoxVE, ...): piVCCU on oDroid C2
    OS (Original, RaspberryMatic, debmatic/...): CCU3
    Version: 3.61.7

**Problem-relevant configuration:**

Do you use tls? No
Do you use callback?0.0.0.0
Do you use username and password? Yes
Which interfaces do you use? HmIP, Bidcos-RF

**Describe the bug**
When trying to set a system variables value via Home Assistant, nothing happens. Home Assistants log is also empty. 

**To Reproduce**
1. Go to 'Developer Tools'
2. Click on 'Services'
3. Scroll down to 'Homematic(IP) local:  Set variable value'
4. Fill it out like in the old integration.
4. See nothing happening

**Expected behavior**
Well, it should just set the variable like the old integration :D

**Screenshots**
![grafik](https://user-images.githubusercontent.com/4162496/149008686-d824172b-d1df-4480-bf91-10340bac53c1.png)
Notice that HA also says successful by marking "Dienst ausführen" green.

**Additional context**
This works(old integration):
```
service: homematic.set_variable_value
data:
  entity_id: homematic.ccu3
  name: Haustür
  value: 'false'
```

This does not:
```
service: hahm.set_variable_value
data:
  entity_id: hahm.pivccu
  name: Haustür
  value: 'false'

```

**Comment by SukramJ** [comment]
Thanks for reporting. Fixed by 0.19.1

**Comment by Lemocin** [comment]
Tested, working! Thanks
